app/views/oAuth/authorize.scala

```diff
@@ -10,23 +10,36 @@ import lila.oauth.AuthorizationRequest
 import lila.oauth.OAuthScope
 
 object authorize {
+
+  val ringsImage = img(
+    cls := "oauth__logo",
+    alt := "linked rings icon",
+    src := assetUrl("images/icons/linked-rings.png")
+  )
+
+  def footer(redirectUrl: String) = div(cls := "oauth__footer")(
+    p(
+      "Not owned or operated by lichess.org"
+    ),
+    p(cls := "oauth__redirect")(
+      "Will redirect to ",
+      redirectUrl
+    )
+  )
+
   def apply(prompt: AuthorizationRequest.Prompt, me: User, authorizeUrl: String)(implicit ctx: Context) =
     views.html.base.layout(
       title = "Authorization",
       moreCss = cssTag("oauth"),
       moreJs = embedJsUnsafe(
         // ensure maximum browser compatibility
-        """setTimeout(function(){var el=document.getElementById('oauth-authorize');el.removeAttribute('disabled');el.setAttribute('class','button')}, 2000);"""
+        """setTimeout(() => {const el=document.getElementById('oauth-authorize');el.removeAttribute('disabled');el.classList.remove('button')}, 2000);"""
```

Comment:
Intentionally going for much broader browser compatibility here, because apps might open the OAuth prompt in ancient web views.